src/pages/home/sidebar/ChatSwitcherSearchForm.js

```diff
@@ -134,6 +134,7 @@ const ChatSwitcherSearchForm = props => (
                 </View>
             )
             : (
+              <View style={styles.textInputContainer}>
```

Comment:
Please fix these lint errors :) We use 4 spaces to indent.

Reply:
Fixed 👍